tests/test_controller.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from rest_framework.test import APIClient
+
+
+class User:
```

Comment:
Move this class to a separate file, if I understand it correctly it will be used in other tests as well.